camille-bot/package.json

```diff
@@ -10,7 +10,8 @@
     "setup-dev-kv": "./scripts/setup-dev-kv.sh",
     "deploy": "wrangler deploy",
     "format": "prettier --write \"src/**/*.ts\"",
-    "test": "jest"
+    "test": "jest",
+    "prod-secrets": "npx wrangler secret bulk .dev.vars"
```

Comment:
quality of life improvement to deploy your vars to prod, could probably  have a .prod.vars file and then just rename the file.